app/locale/de/LC_MESSAGES/django.po

```diff
@@ -2,13 +2,11 @@
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
```

Comment:
I will rebase your PR on `change/improve-user-account-flow` and remove this commit, the .po file has some items that must be handled manually  (because some content is stored into variables I have to add this "manually" at the end of the file). My plan to deal with translation is to focus on it at once in a dedicated PR. It is good to add the `translate` tags in the html though, just no need to update the .po and .mon files

Reply:
> > If we work in a team
> 
> It was not the case until you came ;), the reason I was not merging to often is that I can then quickly update the app online and because I had no pressure since I was alone on the repo :P

Yes, thanks for the clarification. All good :D

Reply:
OK, thanks for the info, I will not touch and po/mo files from now on.